sprint7/test.js

```diff
@@ -0,0 +1,2 @@
+a = undefined;
```

Comment:
Nu cred ca e nevoie de asta in sprint2 :)
Eventual iti recomand:
1. te muti in branchul sprint2
2. apoi stergi acest fisier de acolo
3. apoi scrii git add . 
4. scrii git commit
5. scrii git push
6. si te uiti aici in acest PR: https://github.com/vara-pepene/SkillBrain-Momo/pull/1/files sa vedem ca acest fisier nu se mai afla aici.

Reply:
Multumesc! L-am bagat de seama si nu eram sigura cum sa rezolv pentru ca ma incurcasem intre branch-uri, asa ca nici nu l-am trecut inca in Notion. Il rezolv!